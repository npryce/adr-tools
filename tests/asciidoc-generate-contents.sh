echo "adoc" > .adr-file-type
adr new First Decision
adr new Second Decision
adr new Third Decision
adr generate toc
