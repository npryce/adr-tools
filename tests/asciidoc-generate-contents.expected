echo "adoc" > .adr-file-type
adr new First Decision
doc/adr/0001-first-decision.adoc
adr new Second Decision
doc/adr/0002-second-decision.adoc
adr new Third Decision
doc/adr/0003-third-decision.adoc
adr generate toc
= Architecture Decision Records

* link:0001-first-decision.adoc[1. First Decision]
* link:0002-second-decision.adoc[2. Second Decision]
* link:0003-third-decision.adoc[3. Third Decision]
